src/components/subtitlesettings/subtitleappearancehelper.js

```diff
@@ -28,15 +28,27 @@ function getTextStyles(settings, preview) {
             break;
     }
 
+    switch (settings.textWeight || '') {
+        case 'normal':
+            list.push({ name: 'font-weight', value: 'normal' });
+            break;
+        case 'bold':
+            list.push({ name: 'font-weight', value: '600' });
+            break;
+        default:
+            list.push({ name: 'font-weight', value: 'normal' });
+            break;
+    }
+
```

Comment:
Good start, but save it for later - the release branch is impossible to translate anyway.
Create another branch and cherry-pick the "bold-option" commit, then return to this branch. We will squash commits without it later.

Reply:
56377c7e52b75d7e10bcacdf297d2ee4360ebf31